app/assets/stylesheets/components/_result-sections.scss

```diff
@@ -9,3 +9,20 @@
   @include govuk-responsive-padding(4, "top");
   border-top: 1px solid $govuk-border-colour;
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .app-c-result-sections {
+    margin-bottom: 1cm;
+    padding: 1mm 0;
+  }
+
+  .app-c-result-sections__section  {
+    border-top: 0;
+
+    &:not(:last-of-type) {
+      border-bottom: 1px solid $govuk-border-colour;
```

Comment:
It applies a bottom border to every `.app-c-result-sections__section` except the last one. We don't need a border on the last one because if there's another section that follows, that section starts with a thicker top border. Here's a screenshot which should hopefully make it clearer:

![image](https://github.com/user-attachments/assets/23b75260-eef1-425a-8fce-4e1bef006bf1)

Reply:
That's helpful, thanks, all good.